ecommerce/migrations/0002_productcategory_productsubcategory_delete_category_and_more.py

```
@@ -0,0 +1,102 @@
+# Generated by Django 4.2.23 on 2025-08-05 09:40
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ecommerce', '0001_initial'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)